packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/configmid.md

```diff
@@ -8,11 +8,12 @@
     signOutRedirectUri=<SIGN_OUT_REDIRECT_URI>
     ```
    | Placeholder               | Value                                                                                                                                                        |
-   |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------| ----- |
+   |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
the extra column broke the table. In preview, the table should look the same as in prod
https://64cb2d239d67e0343c74514b--reverent-murdock-829d24.netlify.app/docs/guides/sign-into-mobile-app-redirect/android/main/#configure-your-app
https://developer.okta.com/docs/guides/sign-into-mobile-app-redirect/android/main/#configure-your-app